gnovm/pkg/gnomod/gnomod.go

```diff
@@ -163,6 +165,68 @@ func GnoToGoMod(f File) (*File, error) {
 	return &f, nil
 }
 
+func CreateGnoModFile(rootDir, modPath string) error {
+	if !filepath.IsAbs(rootDir) {
+		return fmt.Errorf("dir %q is not absolute", rootDir)
+	}
+
+	modFilePath := filepath.Join(rootDir, "gno.mod")
+	if _, err := os.Stat(modFilePath); err == nil {
+		return errors.New("gno.mod file already exists")
+	}
+
+	if modPath == "" {
+		// Check .gno files for package name
+		// and use it as modPath
+		files, err := ioutil.ReadDir(rootDir)
+		if err != nil {
+			fmt.Errorf("read dir %q: %w", rootDir, err)
+		}
+
+		var pkgName gnolang.Name
+		for _, file := range files {
+			if file.IsDir() || !strings.HasSuffix(file.Name(), ".gno") || strings.HasSuffix(file.Name(), "_filetest.gno") {
```

Comment:
We should create helpers for this in another PR. Also discussed with @tbruyelle in another PR.

Reply:
Okay. Will open PR for that soon.